### PR TITLE
Added Home page

### PR DESCRIPTION
**Pull request by Amandeep-sinha**
Created Home Page for e-Commerce web-App

please have a look , and give it a review

Screenshot 2022-02-19 151844
image

you can visit it live at :- https://zen-wright-773357.netlify.app/

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,174 @@
+.navigation-bar{
+    gap:60%;
+}
+
+body{
+    background: rgb(224, 220, 220);
```
Comment (siddhigate) [inline style.css:6]: You can CSS variable here for the color.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+}
+.categories{
+    
+    text-align: center;
+    
+}
+
+.categories img{
+    padding: 0;
+}
```
Comment (siddhigate) [inline style.css:16]: You can use a code formatter like Prettier to format the code for consistent spacing, will make your code cleaner

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+</head>
+<body>
+    <!-- Navbar -->
+    <nav class="top-navbar">
+        <div class="top-navbar-icon">
+            <img src="https://bit.ly/3v2WZoY"/>
+            <div class="title">MetaKart</div>
+        </div>
+            <div class="single">
+                <label for="search" style="color: white;  padding: .5rem; " class="fa fa-search fa-2x"></label>
```
Comment (msdeshmukh009) [inline Index.html:19]: Inline styling is not a great idea. Maybe create a class for this one.

File: Index.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                <input type="search" name="search" id="search1" placeholder="Search">
+            </div>
+        
+        <div class="top-navbar-menu">
+            <a href="signin.html" class="top-menu home"><button class="btn">Login</button></a> &nbsp;&nbsp;&nbsp;
+            <a class="top-menu" href="cart.html"><i class="fa fa-shopping-cart fa-2x"><span>2</span></i></a> &nbsp;&nbsp;
+            <a  class="top-menu" href=""><i class="fa fa-heart fa-2x"><span>4</span></i></a>
+            &nbsp;&nbsp;&nbsp;            
+            &nbsp;&nbsp;&nbsp;
```
Comment (msdeshmukh009) [inline Index.html:28]: Any particular reason why you have used HTML entity here `&nbsp;`. Maybe you can do this flex as well.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+
+    .sidebar{
+        height: 250%;
+    }
+
+    .top-navbar{
+        background-color:rgb(83, 223, 223);
+    }
+
+i span{
```
Comment (msdeshmukh009) [inline style.css:94]: It's not really good practice to style HTML elements directly.

File: style.css (hunk 2 above)
Comment (Amandeep-sinha) [reply style.css:16]: Okay siddhi will use that.... Thank u

File: Index.html (hunk 4 above)
Comment (Amandeep-sinha) [reply Index.html:28]: No Mahesh there isn't any specific reason for that..., Yepp i can use flex for the same but this one i found easy that's why gone through it 😅
Thanks, I will do with flex while implementing media queries....

**Pull request merged**